tests/onegov/org/test_browser.py

```diff
@@ -439,3 +440,32 @@ def test_external_map_link(browser, client):
     browser.visit('/topics/themen')
     browser.find_by_css(".button-state").click()
     assert browser.is_text_present("Karte Geo-BS")
+
+
+@pytest.mark.flaky(reruns=3)
```

Comment:
Any idea why this is flaky? Maybe better just use a view test instead of a browser test (if possible) instead of introducing more flakiness...